client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -190,7 +194,8 @@ export const PrescriptionLineEditForm: React.FC<
 
   const handlePrescribedQuantityChange = (inputPrescribedQuantity?: number) => {
     if (inputPrescribedQuantity == null) return;
-    setPrescribedQuantity(inputPrescribedQuantity);
+    if (preferences?.editPrescribedQuantityOnPrescription)
+      setPrescribedQuantity(inputPrescribedQuantity);
```

Comment:
Just wondering if we still need this check since the input calling this function won't even render unless the preference is true

Reply:
Yeah, true. Thanks 207c628ca3